template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Extensions/AzureAdB2C/AzureAdB2COptions.cs

```diff
@@ -1,22 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace Microsoft.AspNetCore.Authentication.Extensions
+﻿namespace Microsoft.AspNetCore.Authentication.Extensions
 {
     public class AzureAdB2COptions
     {
-        public const string PolicyAuthenticationProperty = "Policy";
-
         public string ClientId { get; set; }
-        public string AzureAdB2CInstance { get; set; }
+        public string Instance { get; set; }
```

Comment:
This is just the base URI so that you could potentially point at a staging environment or some other variant.